src/main/java/edu/cornell/kfs/fp/batch/service/impl/CuDisbursementVoucherDocumentGenerator.java

```diff
@@ -167,6 +178,32 @@ protected void validateAndPopulatePayeeTypeCodeAndPayeeNumber(DisbursementVouche
         payeeDetail.setDisbVchrPayeeEmployeeCode(isEmployee);
         payeeDetail.setDisbursementVoucherPayeeTypeCode(convertedPayeeTypeCode);
         payeeDetail.setDisbVchrPayeeIdNumber(payeeId);
+        payeeDetail.setDisbVchrPayeePersonName(vendorPayeeName);
+    }
+    
+    protected void throwPayeeIdValidationError(String errorMessageKey, String payeeId) { 
+        String employeeErrorMessage = configurationService.getPropertyValueAsString(errorMessageKey);
+        throw new ValidationException(MessageFormat.format(employeeErrorMessage, payeeId));
+    }
+    
+    protected void addWarningMEssageIfPayeeNameNotAccurate(String vendorPayeeName, String dvPayeeName) {
+        if (StringUtils.isNotBlank(dvPayeeName) && !StringUtils.equalsIgnoreCase(vendorPayeeName, dvPayeeName)) {
+            AutoPopulatingList<ErrorMessage> warningMessages = new AutoPopulatingList(ErrorMessage.class);
+            String messageString = MessageFormat.format(
+                    configurationService.getPropertyValueAsString(
+                            CuFPKeyConstants.CREATE_ACCOUNTING_DOCUMENT_PAYEE_NAME_NOT_SAME_AS_VENDOR),
+                    vendorPayeeName, dvPayeeName);
+            warningMessages.add(new ErrorMessage(CuFPConstants.CreateAccountingDocumentValidatedDataElements.PAYEE_NAME, messageString));
+
+            LOG.error("addWarningMEssageIfPayeeNameNotAccurate, " + messageString);
```

Comment:
Since the message being printed is a warning message, should this statement log it at the warning level or the error level?